azavea/pfb-network-connectivity
# Add default speed to AnalysisJob model and display on the frontend

## Overview

The default speed limit for a Neighborhood is an important parameter in the BNA analysis, and it needs to be passed along so it can be displayed on the frontend. It is now being exported as part of the analysis process, and then imported into the `default_speed_limit` field on the AnalysisJob model through the `import_default_speed` management command. 

It is then displayed on the frontend in a row in the sidebar table, and as a Leaflet control on the bottom of the map.

### Demo

![image](https://user-images.githubusercontent.com/66973361/98413213-393d0800-2047-11eb-97df-c0d1ec13db90.png)


![image](https://user-images.githubusercontent.com/66973361/95609902-78b70b00-0a2d-11eb-90d7-332c785ecd8f.png)


### Notes


## Testing Instructions

 * Create a Neighborhood, ensuring that it's FIPS code is correct since this is used in the matching of default_speed
 * Run an analysis job for the newly created neighborhood (Boulder shown in screenshots)
 * Validate that default_speed_limit field has been copied to the AnalysisJob in the [Analysis Jobs admin page](http://localhost:9200/#/admin/analysis-jobs/), as shown in the 2nd screenshot
 * Navigate to the Place Detail page. The default speed limit should be displayed in a Leaflet Control in the bottom right next to the map legend.


## Checklist


Resolves #804
